without-webhooks/server/java/src/main/java/com/stripe/sample/Server.java

```diff
@@ -140,18 +140,20 @@ public static void main(String[] args) {
             try {
                 if (postBody.getPaymentIntentId() == null) {
                     int orderAmount = calculateOrderAmount(postBody.getItems());
-                    // Create new PaymentIntent for the order
+                    // Create new PaymentIntent with a payment method ID from the client.
```

Comment:
nit: if we refer to a Stripe resource we should refer to it camelcase like PaymentMethod

Reply:
ack!